linera-core/src/unit_tests/test_utils.rs

```diff
@@ -588,13 +588,19 @@ where
         sender: oneshot::Sender<Result<CryptoHash, NodeError>>,
     ) -> Result<(), Result<CryptoHash, NodeError>> {
         let validator = self.client.lock().await;
-        let certificate_hash = validator
+        let blob_state = validator
             .state
             .storage_client()
             .read_blob_state(blob_id)
             .await
-            .map(|blob_state| blob_state.last_used_by)
             .map_err(Into::into);
+        let certificate_hash = match blob_state {
+            Err(err) => Err(err),
```

Comment:
Since you're returning the error, we can just propagate it with `?` above directly. i.e. `.await?`. Then you shoudnt even need the `map_err(Into::into)` b/c `?` transforms the error automatically IIRC 🤔